docs/public-networks/reference/api/index.md

```diff
@@ -1769,6 +1769,60 @@ None
         }
         ```
 
+### `eth_createAccessList`
+
+Creates a [EIP-2930](https://eips.ethereum.org/EIPS/eip-2930) access list for a transaction.
+
+If your transaction code has already been executed then use the method to generate your access list, otherwise
+send the transaction [along with the `accessList`](../../concepts/transactions/types.md#access_list-transactions) to lower gas costs.
+
+#### Parameters
+
+`transaction`: *object* - [transaction call object](objects.md#transaction-call-object)
+
+`blockNumber` or `blockHash`: *string* - integer representing a block number, block hash, or one of
+the string tags `latest`, `earliest`, or `pending`, as described in
+[block parameter](../../how-to/use-besu-api/json-rpc.md#block-parameter).
+
+#### Returns
+
+`result`: *object* - addresses and storage keys that will be accessed by the transaction:
+
+* `accessList`: *array of objects* - contains the following fields:
+    * `address`: *string* - addresses to be accessed by the transaction
+    * `storageKeys`: *array* - storage keys to be accessed by the transaction
+* `gasUsed`: *string* - approximate gas cost to add the access list to the transaction
```

Comment:
the approximate gas cost that the transaction would actually cost if the access list is included